packages/analytics/metric-cards/src/components/MetricCardContainer.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div class="kong-ui-public-metric-cards">
```

Comment:
Should the wrapper class name be more specific to this individual component?

Reply:
Is `kong-ui-public-metric-card-container` preferable?

Reply:
sure